Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,6 +1,36 @@
-function setAlarm() {}
+function setAlarm() {
+  const inputField = document.getElementById("alarmSet");
+  const timeRemaining = document.getElementById("timeRemaining");
+  let timeLeft = parseInt(inputField.value);
 
-// DO NOT EDIT BELOW HERE
+  function updateTimerDisplay(seconds) {
+    const minutes = Math.floor(seconds / 60);
+    const secs = seconds % 60;
+    timeRemaining.textContent = `Time Remaining: ${String(minutes).padStart(
+      2,
+      "0"
+    )}:${String(secs).padStart(2, "0")}`;
+  }
+
+  updateTimerDisplay(timeLeft);
+
+  const countdown = setInterval(() => {
+    timeLeft--;
+
+    if (timeLeft <= 0) {
+      clearInterval(countdown);
+      timeLeft = 0;
+      playAlarm();
+    }
+
+    updateTimerDisplay(timeLeft);
+  }, 1000);
+
+  document.getElementById("stop").addEventListener("click", () => {
+    clearInterval(countdown);
+    pauseAlarm();
+  });
+}
```

Comment:
Hey @Forogh-Aghaeiyarijani , good effort. 
After checking your code and HTML , it does not show me correct time remaining. 
It should show you what is remaining after you add a number in your input. please fix this and request me to review it